doc/specifications/cache/repository_cache_tags.md

```diff
@@ -0,0 +1,70 @@
+## Tags
+
+As the system is extensible, important part of this is to document the used tags to avoid wrong use, or conflicts.
+If you add own tag types, please prefix with 1-3 letters abbreviating your *(company/full)* name.
+
+The tags needs to be made in a way so that cache can be cleared using nothing but what info is available on signal,
+and the signals will need to be expanded to contain the relevant info depending on operation they correspond to.
+
+#### Content Tags
+
+
+Tags applied to Content View and Content REST Object:
+
+- `content-<content-id>`
+
+    *Tagging*: Used for tagging content responses with id.
+    
+    *Clearing*: When a operation is specifically affecting just given content.
+
+- `content-type-<content-type-id>`
+
+    *Tagging*: Used for tagging content responses with type id.
+    
+    *Clearing*: When a operation is specifically affecting content type, typically soft purge all affected content.
+
+#### Location and Content Location Tags
+
+
+- `location-<location-id>`
+
+    If content has locations we need additional tags on the content response to be able to clear on operations affecting a
+    given location or a tree.
+    
+    *Tagging*: Used for tagging content responses with all it's locations.
+    
+    *Clearing*: When a operation is specifically affecting one or several locations, on tree operations `path` is more relevant.
+
+
+- `parent-<parent-location-id>`
+
+    *Tagging*: Used for tagging content responses with all it's direct parents.
+    
+    *Clearing*: When a operation is specifically affecting parent location(s), on tree operations `path` is more relevant.
+
+- `path-<path-location-id>`
```

Comment:
we'll need to expand here, and maybe update doc to reflect the code.

Side, we could also consider aligning with *(one or the other way)*: https://github.com/ezsystems/ezpublish-kernel/compare/sf_cache#diff-e7db5fd1ab0e388fd87f6a8095599e22R277 and https://github.com/ezsystems/ezpublish-kernel/compare/sf_cache#diff-fb693bf163e578a01367f4466a44ad9bR316